portfolio/src/main/webapp/script.js

```diff
@@ -336,9 +337,24 @@ async function addMarker(position) {
     displayMarkers();
 }
 
+async function checkStatus() {
+    const response = await fetch('/login/status');
+    const status = await response.text();
+    if (status.trim().localeCompare("true") === 0) {
+        document.getElementById("comments-login").style.display="none";
+        document.getElementById("comments-logout").style.display="block";
+        document.getElementById("comments-form").style.display="block";
+        fetchCommentsWithStoredLimit();
+    } else {
+        document.getElementById("comments-login").style.display="block";
+        document.getElementById("comments-logout").style.display="none";
+        document.getElementById("comments-form").style.display="none";
```

Comment:
Should have whitespace around the `=` in these statements.

Reply:
I forgot to add the whitespaces when I typed first line and then copy paste the left 5 lines. I will add them in the newest commit.